### PR TITLE
#521: Added CI docs on how to add a test and how CI infrastructure works

### PR DESCRIPTION
**Pull request by tapspatel**

### CODE REVIEW

File: docs/src/ci.md
Comment (nsmithtt) [inline docs/src/ci.md:1]: Maybe I'm missing something, but how can we navigate to this page? I think we need to reference it from SUMMARY.md

Comment (tapspatel) [reply docs/src/ci.md:1]: fixed

File: docs/src/ci.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# `ci`
+
+Our CI infrastructure is currently hoisted on cloud. We use cloud machines and link them as GitHub runners.
```
Comment (vmilosevic) [inline docs/src/ci.md:3]: hoisted -> hosted

**Review by vmilosevic — APPROVED** [review]

File: docs/src/ci.md (hunk 1 above)
Comment (tapspatel) [reply docs/src/ci.md]: fixed

**Review by nsmithtt — APPROVED** [review]

**Pull request merged**